### PR TITLE
Ecs release fix

### PR DESCRIPTION
**Pull request by rajeevr2715**
Jenkins ecs resource renaming
Allow taggings in ecs
Reduce the Memory and CPU

### CODE REVIEW

File: installscripts/jazz-terraform-unix-noinstances/ecs_jenkins_task_definition.json [hunk 1]
```diff
@@ -19,9 +19,9 @@
           "value": "${jenkins_user}"
         }
       ],
-    "memoryReservation": 2048,
+    "memoryReservation": 3072,
     "networkMode": "awsvpc",
-    "cpu": 2048,
+    "cpu": 1024,
```
Comment (suryajak) [inline installscripts/jazz-terraform-unix-noinstances/ecs_jenkins_task_definition.json]: Shouldn't this be reading off of `variables.tf` as opposed to they being hard coded?

File: installscripts/jazz-terraform-unix-noinstances/ecs_jenkins_task_definition.json [hunk 2]
```diff
@@ -19,9 +19,9 @@
           "value": "${jenkins_user}"
         }
       ],
-    "memoryReservation": 2048,
+    "memoryReservation": 3072,
```
Comment (suryajak) [inline installscripts/jazz-terraform-unix-noinstances/ecs_jenkins_task_definition.json]: Same as earlier.

File: installscripts/jazz-terraform-unix-noinstances/ecs_jenkins_task_definition.json (hunk 1 above)
Comment (rajeevr2715) [reply installscripts/jazz-terraform-unix-noinstances/ecs_jenkins_task_definition.json]: Done.

File: installscripts/jazz-terraform-unix-noinstances/ecs_jenkins_task_definition.json (hunk 2 above)
Comment (rajeevr2715) [reply installscripts/jazz-terraform-unix-noinstances/ecs_jenkins_task_definition.json]: Done.

**Pull request merged**